docs/data-and-backup.md

```diff
@@ -1 +1,98 @@
 # Data and Backup
+
+## Data Storage Location
+
+The location where mhira-docker installation will store its data is controlled by environment variable `DATA_SAVE_PATH`.
+
+```dotenv
+# Default location under current user home directory
+DATA_SAVE_PATH=~/.mhira-docker/data
+```
+
+This is where all stateful containers (containers which persist data) will store their configuration and data.
```

Comment:
wouldn't you need to use the word 'docker volumes' here?
Maybe give an example?

Reply:
For mhira case, the container persistance mechanism uses "bind-mounts" rather than "volumes" so it not applicable for this case. The difference expanded [here on stackoverflow](https://stackoverflow.com/questions/34357252/docker-data-volume-vs-mounted-host-directory)